redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -34,5 +34,36 @@ public function __construct()
         }
 
         $this->PluginDir = $addon->getPath('lib/phpmailer/');
+        
+        parent::__construct($exceptions);
+    }
+    
+    public function send()
+    {
+        $this->backup();
+        return parent::send();
+    }
+    
+    private function backup()
+    {
+        $content = '<!-- '.PHP_EOL.date('d.m.Y H:i:s').PHP_EOL;
+        $content .= 'From : '.$this->From.PHP_EOL;
+        $content .= 'To : '.implode(', ', array_column($this->getToAddresses(), 0)).PHP_EOL;
+        $content .= 'Subject : '.$this->Subject.PHP_EOL;
+        $content .= ' -->'.PHP_EOL;
+        $content .= $this->Body;
+        
+        $dir = rex_path::frontend().'mail_backup';
```

Comment:
bitte in den data-dir des addons und nicht ins frontend schreiben. dann brauchts auch keine zusätzliche .htaccess